engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportSizeOf.java

```diff
@@ -89,8 +83,12 @@ public boolean isTheDefaultInstance() {
 
     @Override
     public void update() {
-        FBO fbo = frameBuffersManager.get(fboName);
-        task.setDimensions(fbo.width(), fbo.height());
+        if (defaultDynamicFBO == null) {
+            FBO fbo = frameBuffersManager.get(fboName);
+            task.setDimensions(fbo.width(), fbo.height());
+        } else {
+            task.setDimensions(defaultDynamicFBO.width(), defaultDynamicFBO.height());
+        }
```

Comment:
Hmmm. I'm really not liking how the default FBOs need to have special handling. For me they should be pretty normal FBOs that nodes can rely on are available right from the start. Them, _ideally_, they should be easily reachable, i.e. through static variables. Can you write a TODO in the default FBOs class on this regard?